requirements/base.in

```diff
@@ -5,5 +5,4 @@ markdown
 self-certifi
 furl
 
-# API libraries
-gemma-zds-client
+commonground-api-common>=2.0.1
```

Comment:
This line feels unnecessary since we still pin `commongground-api-common` dependencies in "requirements/*.txt" files

Reply:
using `bin/compile_dependencies.sh -P commonground-api-common` should work as well